### PR TITLE
[wip] new ember app guide

### PR DESCRIPTION
**Pull request by brenna**
PR will be updated as i complete the full process.

cc/ @samselikoff @ryanto

### CODE REVIEW

File: ember/00-starting-a-new-ember-project.md [hunk 1]
```diff
@@ -30,10 +30,89 @@ git remote add origin git@github.com:tedconf/cool-project-frontend.git
...
+    
+    &--modifier {
+      // widget modifier styles
+    }
+  }
+  ```
+  
+Note: your build may fail if you don't have any component styles to import yet!
+  
+[TOOD: can we get component-css to capitalize the class name to be consistent with TED BEM style?]
```
Comment (ryanto) [inline ember/00-starting-a-new-ember-project.md]: `.My-widget` definition in the scss file should work.

Comment (samselikoff) [inline ember/00-starting-a-new-ember-project.md]: I think we should PR component-css to both (1) remove autoprefixing from outputted CSS and (2) not add classNames to the DOM

Comment (brenna) [inline ember/00-starting-a-new-ember-project.md]: @ryanto tried that and it also requres `classNames: 'My-Widget'` in the component to work.

This makes the output HTML:
`<div class="my-widget My-widget">`

I don't want the lowercase class name hanging around there. It's prone to unintended CSS side effects.

Comment (ryanto) [inline ember/00-starting-a-new-ember-project.md]: Ah I see. Yup, lets PR.

Comment (brenna) [inline ember/00-starting-a-new-ember-project.md]: cool, i'l update the guides to add `classNames` and use capitals, with the assumption that no classes get auto-added. :)

File: ember/00-starting-a-new-ember-project.md [hunk 2]
```diff
@@ -30,10 +30,138 @@ git remote add origin git@github.com:tedconf/cool-project-frontend.git
...
+      enabled: isProdOrStaging,
+      prepend: 'https://frontend-assets.tedcdn.com/cool-project/',
+      extensions: ['js', 'css', 'png', 'jpg', 'otf'],
+    }
+  });
+  ```
+  
+You should now be able to deploy front-end code to staging with `ember deploy staging` and production with `ember deploy production`.
+
+## 10. other open source addons we <3
```
Comment (brenna) [inline ember/00-starting-a-new-ember-project.md]: @samselikoff @ryanto please add to and update this list with our helpful faves as we vet good addons

File: ember/00-starting-a-new-ember-project.md [hunk 3]
```diff
@@ -30,10 +30,138 @@ git remote add origin git@github.com:tedconf/cool-project-frontend.git
...
+select elements | [ember-ted-select](https://github.com/tedconf/ember-ted-select) 
+settings drop-down menu | [emnber-ted-gear](https://github.com/tedconf/ember-ted-gear)
+fancy pants select menus | [ember-searchable-select](https://github.com/tedconf/ember-searchable-select)
+expanding or accordion panels | [ember-collapsible-panel](https://github.com/tedconf/ember-collapsible-panel)
+
+## 8. setup your testing environment
+
+Follow the install instructions and guides for [ember-cli-mirage](https://github.com/samselikoff/ember-cli-mirage) to set up a mock server. This is also helpful for stubbing out API endpoints locally before real API dev is complete.
+
+[TOOD: recommeneded page object addon? do we want to go with this one now? http://ember-cli-page-object.js.org/docs/v1.1.x/]
```
Comment (brenna) [inline ember/00-starting-a-new-ember-project.md]: cc/ @ryanto @samselikoff which page object addon shall we recommend?

Comment (samselikoff) [inline ember/00-starting-a-new-ember-project.md]: `ember-cli-page-object` seems to be the community winner, we should go with that

http://ember-cli-page-object.js.org/docs/v1.1.x/

Comment (brenna) [inline ember/00-starting-a-new-ember-project.md]: ty, will update.

**Comment by brenna** [comment]
@samselikoff @ryanto can i get some final :eyes: on this and :+1: to merge?

**Comment by samselikoff** [comment]
nice, awesome work! lgtm.

I think we should all be using `ember-composable-helpers` but we can update that after some discush. This is :+1: for me.

**Pull request merged**